ipn/ipnlocal/peerapi_h2c.go

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-//go:build !ios && !android
-// +build !ios,!android
+//go:build !ios && !android && !js
```

Comment:
what, you don't want to run an Exit Node in a browser and have good DNS proxy throughput? 😂